app/lib/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+// in Wii mode, enable `no_std` and import ogc_rs replacements of common functions.
+#![cfg_attr(feature = "wii", no_std)]
+#[cfg(feature = "wii")]
+pub use ogc_rs::{print, println};
```

Comment:
Is this change compatible with https://github.com/snappy-cobra/WiiDevEnv/pull/25 ? Or is it a replacement?

Reply:
It is compatible. 

This PR enables us to use 'normal' Rust unit tests. The limitation is that we cannot call `ogc_rs`/`grrrlib` or other Wii-specific stuff in those tests (or in the `lib` crate as a whole). Functions that are available in `std` and re-implemented in `ogc_rs` like `println` are of course fine!
Because Rust's builtin test runner and native compilation is used, we have access to many of the nicer testing features, such as:
- Backtraces on failure
- Show all test failures rather than only the first one
- Editor integration
- Doctests
- Tests can live close to the code they are testing. (In #25 they need to manually be added to the 'hasmap of tests')
- The possibility to filter tests when testing something specific.
- Testing for `should_panic` and similar.
- etc.

#25 enables on-target integration tests. Here the full Wii with all calls in `ogc_rs`/`grrrlib` is available. But those nice abstractions listed above are not.

Reply:
Cool, makes a lot of sense and very nice to have this :+1: